x-pack/platform/plugins/shared/inference/server/plugin.ts

```diff
@@ -109,6 +111,13 @@ export class InferencePlugin
           logger: this.logger,
         });
       },
+
+      getConnectorList: async (request: KibanaRequest) => {
+        return getConnectorList({ actions: pluginsStart.actions, request });
+      },
+      getDefaultConnector: async (request: KibanaRequest) => {
```

Comment:
Not part of the original issue, but we also have a util getConnectorById https://github.com/elastic/kibana/blame/main/x-pack/platform/plugins/shared/inference/server/util/get_connector_by_id.ts/#L19 that I think would be useful to expose here. Is it possible?

Reply:
Updated in https://github.com/elastic/kibana/pull/230172/commits/833b04d99f1233ef2da23422f3408e8d57c63ad1